build/images/base/build_and_push.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a very simple script that builds the base image for Antrea and pushes it to
+# the Antrea Dockerhub (https://hub.docker.com/u/antrea). The image is tagged with the OVS version.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+if [ -z "$OVS_VERSION" ]; then
+    echoerr "The OVS_VERSION env variable must be set to a valid value (e.g. 2.13.0)"
+    exit 1
+fi
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pushd $THIS_DIR > /dev/null
+
+docker pull ubuntu:18.04
+
+docker build \
+       --cache-from antrea/base:$OVS_VERSION \
```

Comment:
I think you can omit `--cache-from antrea/base:$OVS_VERSION` here. For the ovs image it is meant to avoid the long [build](https://andrewlock.net/caching-docker-layers-on-serverless-build-hosts-with-multi-stage-builds---target,-and---cache-from/#:~:text=The%20%2D%2Dcache%2Dfrom%20argument%20allows%20you%20to%20tell%20Docker,built%20on%20the%20same%20machine.) when unnecessary but here the image builds very fast. If you do want to use the cache, you will additional steps because it is a multi-stage build (unless buildkit is enabled).

Reply:
Good to know, removed.